packages/trader/src/sass/app/modules/settings.scss

```diff
@@ -127,6 +127,9 @@
 .dc-modal-header {
```

Comment:
Please revert this.

Reply:
@aaron-binary no these two casheir and setting i think we need them cause we removed it from modal.scss